release-notes/cabal-install-3.10.1.0.md

```diff
@@ -0,0 +1,227 @@
+    This warning will no longer be emitted for special modes of operation like `cabal --numeric-version`, `--version` and help,
+    and also not if `cabal` could not parse its command line.
+    Further, the warning now goes to `stderr` rather than `stdout`.
+
+- Fix and improve list parser of cabal init cli [#8659](https://github.com/haskell/cabal/issues/8659) [#8663](https://github.com/haskell/cabal/pull/8663)
+
+  Occurrences of 'Flag [a]' seem to behave in an unexpected way. The monoid
+  instance of 'Flag' is right associative and discard the value on the
+  left, but we want to merge the contents of 'Flag'.
+
+  Permits:
+  - cabal init -d base -d vector -d containers
+
+  Fixes for all Flag '[a]' the cli parser in cabal init. Adds cli parser tests.
+
+  Adds the feature to specify a comma-separated list of dependencies:
+  - cabal init -d base,vector,containers
+
+- Fix resolution of imports by relative paths in cabal.project [#8686](https://github.com/haskell/cabal/pull/8686)
+
+  Fix bug where cabal tries to resolve imports by relative paths against
+  the directory cabal executable was invoked in rather than directory of
+  cabal.project file that does the import.
+
+- Add warning for running cabal run, cabal test and cabal bench with +RTS flag [#8709](https://github.com/haskell/cabal/pull/8709)
+
+  This adds a warning when RTS options are passed to cabal instead of the binary
+  for the commands 'run', 'bench' and 'test', as most users want to pass these
+  options to their binary.
+
+- Fix repl discarding --build-depends [#6859](https://github.com/haskell/cabal/issues/6859) [#7081](https://github.com/haskell/cabal/issues/7081) [#8732](https://github.com/haskell/cabal/pull/8732)
+
+  - Fix `repl` command discarding `--build-depends` argument when using
+    `allow-newer` or `allow-older`.
+
+- Add `cabal get --only-package-description` [#1954](https://github.com/haskell/cabal/issues/1954) [#1977](https://github.com/haskell/cabal/pull/1977) [#5162](https://github.com/haskell/cabal/pull/5162) [#8263](https://github.com/haskell/cabal/pull/8263)
+
+- `cabal haddock` now implies `--enable-documentation` [#7462](https://github.com/haskell/cabal/issues/7462) [#8259](https://github.com/haskell/cabal/pull/8259)
+
+- cabal init: suggest BSD-3 as default license [#8757](https://github.com/haskell/cabal/issues/8757) [#8764](https://github.com/haskell/cabal/pull/8764)
+
+- Include extra-lib-dirs-static into PackageHash [#6935](https://github.com/haskell/cabal/issues/6935) [#7794](https://github.com/haskell/cabal/pull/7794)
+
+- Document `cabal check` in the user guide [#8237](https://github.com/haskell/cabal/pull/8237)
+
+- Add documentation for cabal-install commands: user-config, [#7884](https://github.com/haskell/cabal/issues/7884) [#7978](https://github.com/haskell/cabal/issues/7978) [#8308](https://github.com/haskell/cabal/issues/8308) [#8309](https://github.com/haskell/cabal/pull/8309)
```

Comment:
Why the comma after user-config?

Reply:
Hah, apparently the different PRs concern different other commands, so there must be more. Let me cheat and add "and other commands".

Reply:
Done. And it says it's only about cabal-install, but is mentioned also in the Cabal changelog, so I removed it from there.